composer.json

```diff
@@ -1,20 +1,24 @@
 {
     "name": "laminas/laminas-mvc-plugins",
-    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
     "type": "metapackage",
-    "license": "BSD-3-Clause",
+    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
     "keywords": [
         "laminas",
         "mvc",
         "plugins"
     ],
     "homepage": "https://laminas.dev",
-    "support": {
-        "issues": "https://github.com/laminas/laminas-mvc-plugins/issues",
-        "source": "https://github.com/laminas/laminas-mvc-plugins",
-        "rss": "https://github.com/laminas/laminas-mvc-plugins/releases.atom",
-        "chat": "https://laminas.dev/chat",
-        "forum": "https://discourse.laminas.dev"
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": "^7.3 || ~8.0.0",
+        "laminas/laminas-mvc-plugin-fileprg": "^1.2",
+        "laminas/laminas-mvc-plugin-flashmessenger": "^1.3",
+        "laminas/laminas-mvc-plugin-identity": "^1.2",
+        "laminas/laminas-mvc-plugin-prg": "^1.3",
+        "laminas/laminas-zendframework-bridge": "^1.0"
```

Comment:
Also for laminas-zendframework-bridge.

Reply:
As there is no code in this repo, dropping the bridge should be finde as well.

Reply:
```
php8.0 composer.phar why laminas/laminas-zendframework-bridge
[...]
laminas/laminas-mvc-plugin-fileprg         1.2.0   requires  laminas/laminas-zendframework-bridge (^1.0)
laminas/laminas-mvc-plugin-flashmessenger  1.3.0   requires  laminas/laminas-zendframework-bridge (^1.0)
laminas/laminas-mvc-plugin-identity        1.2.0   requires  laminas/laminas-zendframework-bridge (^1.0)
laminas/laminas-mvc-plugin-prg             1.3.0   requires  laminas/laminas-zendframework-bridge (^1.0)
[...]
```